models/db.js

```diff
@@ -24,4 +25,12 @@ db.public.assignments.belongsTo(db.public.login, { foreignKey: 'user_id', onDele
 db.public.login.hasMany(db.public.classes, { foreignKey: 'login_id', onDelete: 'CASCADE', constraints: false, allowNull: false })
 db.public.classes.belongsTo(db.public.login, { foreignKey: 'login_id', onDelete: 'CASCADE', constraints: false, allowNull: false });
 db.public.classes.belongsTo(db.public.subjects, { foreignKey: 'subject_id', onDelete: 'CASCADE', constraints: false, allowNull: false });
+
+//For making the relations on the local database.
+/*sequelize.sync({ force: true })
+  .then(() => {
+    console.log(`Database & tables created!`)
+  })  
+*/
+
```

Comment:
Unnecessary comment

Reply:
called this function to initialize tables in Postgres